interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
+owner @{HOME}/.config/gtk-2.0/gtkfilechooser.ini w,
+owner @{HOME}/.config/gtk-3.0/bookmarks w,
+owner @{HOME}/.config/gtk-3.0/settings.ini w,
```

Comment:
In my mind, that would be a different interface. unity7-settings-control or unity7-control or something. Please use read for now since most applications will only need to read the global settings.